modules/packages/Sort.chpl

```diff
@@ -263,14 +433,15 @@ proc QuickSort(Data: [?Dom] ?eltType, minlen=16, doublecheck=false, param revers
   Data(loptr) = pivotVal;
 
   //  cobegin {
```

Comment:
Why is this Cobegin commented out?
I think it would be worthwhile to allow the user to choose whether the sort is parallel.